coder_dojo_common/settings.py

```diff
@@ -0,0 +1,27 @@
+from pathlib import Path
```

Comment:
Why do we need this `settings.py`?

Reply:
It acts as base settings for everything in the package - where we stored cached assets, etc...